usecases/schema/helpers_for_test.go

```diff
@@ -40,22 +40,22 @@ import (
 func newTestHandler(t *testing.T, db store.DB) (*Handler, func()) {
 	cfg := config.Config{}
 	srv := startRaftCluster(t, db)
-	reader := srv.SchemaReader()
+	metaStore := srv.GetClusterMetaStore()
```

Comment:
The `schema package` is not directly dependent on the `cloud package`
However, it relies on the `metaReader and metaWriter interfaces`, which are internal to the package itself.
Therefore, providing mock implementation of both interfaces is sufficient for unit testing of the schema package